src/webextension/list/manage/components/account-summary.css

```diff
@@ -109,12 +109,33 @@
   list-style-type: none;
 }
 
-.dropdown > ul > li {
-  padding: 8px;
-}
-
 .dropdown-link {
   color: #0c0c0d;
+  display: block;
+  width: 100%;
+  text-align: left;
+  padding: 8px 16px;
+  border-radius: 0;
+  transition: background .15s ease;
```

Comment:
Where's this coming from? From what I can see, the Zeplin mockup doesn't have hover states*, and moreover, the Photon spec doesn't appear to animate hover states for any of the widgets in the spec. If we *did* want to use an animation here (and I'm not sure we do), I think we'd also want to use the Photon animation curve: https://design.firefox.com/photon/motion/duration-and-easing.html

\* One of the many issues I've had with Zeplin...

Reply:
Also, a minor point that you don't have to fix, but we're making these widgets look a lot like buttons. I wonder if we should switch to a `<Button/>` element in the JS (which would give us the "button" ARIA role for free). @linuxwolf Thoughts?

Reply:
@jimporter @nickbrandt  @kimberlythegeek 

It's not clear what the full benefits of `role="button"` over `role="link"` are in this case.  I'm inclined to leave the PR as is for now, and address in the future as needed.